### DISCUSSION TITLE
how can i use Dynamic repository feature with gitversion.MsBuild task

### DISCUSSION THREAD

**Discussion by RDUser-dev** [GitTools/GitVersion#3040] [Q&A] [answered]
Hi, 
I'm using gitversion.MsBuild in a jenkins server. 
I would like to start using the Dynamic repository feature.
the documentation page [dynamic-repositories](https://gitversion.net/docs/learn/dynamic-repositories) explains how to configure the feature for **GitVersion commandline**, but not for **GitVersion.MsBuild**
How can i pass the requirement parameters to **GitVersion.MsBuild** task?

**Comment by asbjornu** [comment, marked as answer]
Only some of the command line arguments are available through MSBuild. #3022 tries to change that, but it is not yet completed.

**Reply by RDUser-dev** [reply]
Thanks @Asbjornu, I think feature #3022 is very important to make GitVersion.MSBuild usable. I hope to see it completed soon. If I can be of any help to @professor-k, don't hesitate to ask

**Reply by RDUser-dev** [reply]
> Only some of the command line arguments are available through MSBuild. #3022 tries to change that, but it is not yet completed.

Hello everyone,
is there any news on the matter?

**Reply by asbjornu** [reply]
I just merged #3022, which will become available in our next 5.x release.
